pkg/agreementreceiver/server.go

```diff
@@ -51,16 +85,17 @@ func (s *Server) BandwidthAgreements(stream pb.Bandwidth_BandwidthAgreementsServ
 		case err := <-errch:
 			return err
 		case <-ctx.Done():
+			s.logger.Debug("ctx<-Done()")
```

Comment:
Needs a better log message to indicate what is done

Reply:
for now as it serves no useful purpose.It is there for any clean up action if needed, then it has to be done here like handling Ctrl+c kind of things